Add fillFUV function #20

Eimutt opened the pull request:
Added function to fill the FUV

Eimutt committed:
Add fillFUV function #20

Added function to fill the FUV

JulienVig commented on Main.java:
```diff
@@ -23,4 +23,21 @@ public static void main(String[] args){
+        for(int i = 0; i < 15; i++){
+            if(!PUV[i]){ // If PUV[i] is false PUM[i] doesn't matter, FUV[i] is always true
+                FUV[i] = true;
+                continue;
```
I don't think the `continue`here is needed since there is an `else`afterward

JulienVig commented:
We still need to test it before merging it to the master branch!

JulienVig commented on Main.java:
```diff
@@ -23,4 +23,21 @@ public static void main(String[] args){
+            if(!PUV[i]){ // If PUV[i] is false PUM[i] doesn't matter, FUV[i] is always true
+                FUV[i] = true;
+                continue;
+            } else { //otherwise makes sure everything in PUM[i] is true
```
I think we need to set the default value to `true` otherwise we never do it when the whole row of the PUM is `true`.
```suggestion
            } else { //otherwise makes sure everything in PUM[i] is true
            FUV[i] = true;
```

JulienVig reviewed (changes requested):
I think the current function isn't working but I'm currently writing tests for it.

JulienVig commented on Main.java:
```diff
@@ -23,4 +23,21 @@ public static void main(String[] args){
+                FUV[i] = true;
+                continue;
+            } else { //otherwise makes sure everything in PUM[i] is true
+                for(int j = 0; j < 14; j++){
```
The indix `j` goes from 0 to 14 so the for loop should have the condition `j<15`
```suggestion
                for(int j = 0; j < 15; j++){
```

JulienVig reviewed (changes requested):
In addition to the setting the default value of the FUV to `true` the second for loop should go all the way to 15.

JulienVig committed:
Add test for the fillFUV function #42

The fillFUV tests whether the FUV is filled according to the PUV and the PUM.
The current tests pass only if we commit the current changes of the pull request i.e.
setting FUV[i] to true as a default value and letting j go all the way to 15.
Furthermore I changed the attributes of the Main class and its methods to public so
that we have access to them from the Tests class.

Eimutt committed:
Update Main.java

Co-Authored-By: JulienVig <33122365+JulienVig@users.noreply.github.com>

Eimutt committed:
Update Main.java

Co-Authored-By: JulienVig <33122365+JulienVig@users.noreply.github.com>
